app/LinkLiteAgent/README.md

```diff
@@ -23,6 +23,22 @@ poetry run linkliteagent
 ```
 and use `Ctrl+C` to shut it down.
 
+### DB for logging
+To build the DB for logging, you will need a running db server (currently only postgresql is supported). At the root of the repo run:
+```shell
+docker-compose up -d
+```
+and you will get a fresh postgresql db.
+
+Then in `app/LinkLiteAgent`, run:
+```shell
```

Comment:
Can we note on this that it's just a backup for working on the agent without the manager?

In normal circumstances, I'd expect the Manager to be considered the "owner" of the config data, and it will have migrations for constructing said database, including the Logs table.

Reply:
How's looking now @beforan?

Reply:
noice